conf/genomes.config

```diff
@@ -15,6 +15,7 @@ params {
             fai                 = "${params.local_genomes}/grch37_homo_sapiens_-d5-.fasta.fai"
             bwamem2             = "${params.local_genomes}/grch37_homo_sapiens_-d5-.fasta.{0123,amb,ann,bwt.2bit.64,pac}"
             gnomad              = "${params.local_genomes}/grch37_gnomad_reformated_-r2.1.1-.vcf.gz"
+            sentieon            = ""
```

Comment:
Is there particular indexes for sentieon, I thought they were the same as bwa

Reply:
I am not sure, but raredisease doesn't use bwa. So I call the parameter "sentieon" mainly to avoid confusion.